proto/waves/node/grpc/accounts_api.proto

```diff
@@ -57,3 +57,12 @@ message ScriptData {
     string script_text = 2;
     int64 complexity = 3;
 }
+
+message LeaseResponse {
+    bytes leaseId = 1;
+    bytes originTransactionId = 2;
+    bytes sender = 3;
+    bytes recipient = 4;
+    int64 amount = 5;
+    int32 height = 6;
```

Comment:
In `TransactionResponse` we have `height` as `int64` but here it's `int32`. Which is correct? https://github.com/wavesplatform/protobuf-schemas/blob/ac1aeb7debd95017ae5a52a28071d6988f61b644/proto/waves/node/grpc/transactions_api.proto#L44

Reply:
Done